### DISCUSSION TITLE
How to reference a table that is not public

### DISCUSSION THREAD

**Discussion by jean-smaug** [prisma/prisma#9128] [Q&A]
I'm using Supabase and I try to use Prisma to handle migrations.

Supabase is using an `auth.users` table for [user management](https://supabase.io/docs/guides/auth#how-it-works)

I would like to create a `profiles` table to enhance data of users.
I would  like profiles id to be the same as users id.
So I would like to reference the id of users on profiles.

```bash
model Profile {
  // This doesn't work
  id     String @id @db.Uuid @relation(name: "auth.users", references)
}
```

Is there any way to reference a table that is not defined in Prisma model ?

**Comment by ryands17** [comment]
@jean-smaug 👋 

This currently isn't possible with Prisma as it supports only a single schema. You can add a 👍 and follow this request for more updates: https://github.com/prisma/prisma/issues/1122

**Reply by jean-smaug** [reply]
Hi,
Thanks for info.

Is there any workaround ?

I mean, I saw on the doc that we could make "raw queries" using client SDK.
Is there any way to manually add contraints after a `prisma db push`.
Does something like a "post-migration" hook exists ?

Thanks

**Reply by ryands17** [reply]
Yes you can query for items in a different schema using `prisma.$queryRaw`, but you cannot add constraints. Also `prisma db push` cannot add constraints as it doesn't generate migrations, only `prisma migrate dev` can.

**Comment by NanoCode012** [comment]
Hey @jean-smaug , would it be possible to share how you solved this issue?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
